Manager/ModuleManager.php

```diff
@@ -126,6 +127,19 @@ public function enableModules($modules)
     }
 
     /**
+     * Delete the given module
+     * @param Module $module
+     */
+    public function deleteModule(Module $module)
+    {
+        $coreModules = $this->getCoreModules();
+        if (!isset($coreModules[$module->getLowerName()])) {
```

Comment:
There is already a check for code module in the controller, this one shouldn't be needed.
Though it is still good to have that security, in that case I would inverse the condition:

``` php
if (isset($coreModules[$module->getLowerName()])) {
    return;
}
$this->module->delete($module);
event(new ModuleHasBeenDeleted($module));
```